cumulusci/tasks/metadeploy.py

```diff
@@ -112,10 +113,13 @@ def _run_task(self):
             version = self._find_or_create_version(product)
             if self.labels_path and "slug" in product:
                 self._publish_labels(product["slug"])
+        else:
+            version = {"id": "fake_for_dryrun", "url": "fake_for_dryrun"}
```

Comment:
This sets up a variable that will never be read because otherwise it would have caused a crash in the old code. So it should be backwards compatible.